policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/weeks_per_year.yaml

```diff
@@ -0,0 +1,10 @@
+description: The Department for Education provides extended childcare entitlement for specified weeks per year.
```

Comment:
```suggestion
description: The Department for Education provides extended childcare entitlement for up to this number of weeks per year.
```

Reply:
Well, the weeks aren't freely movable, they're in term times, so the original is technically more correct